frontend/app/components/company-uebersicht-edit.js

```diff
@@ -1,64 +0,0 @@
-import { inject as service } from '@ember/service';
-import Component from '@ember/component';
-
-export default Component.extend({
-  store: service(),
-  i18n: service(),
-  router: service(),
-
-  actions: {
-    submit(changeset) {
-      changeset.save()
-        .then(() => this.sendAction('submit'))
-        .then(() => this.get('notify').success('Firmenübersicht wurde aktualisiert!'))
-        .then(function() {
-          changeset.get('locations').filterBy('hasDirtyAttributes').toArray().forEach(function(location) {
-            location.save();
-          });
-          changeset.get('employeeQuantities')
-            .filterBy('hasDirtyAttributes').toArray().forEach(function(employeeQuantity) {
-              employeeQuantity.save();
-            });
-        })
-        .catch(() => {
-          let company = this.get('company');
-          let errors = company.get('errors').slice(); // clone array as rollbackAttributes mutates
-          company.rollbackAttributes();
-          errors.forEach(({ attribute, message }) => {
-            let translated_attribute = this.get('i18n').t(`company.${attribute}`)['string']
-            changeset.pushErrors(attribute, message);
-            this.get('notify').alert(`${translated_attribute} ${message}`, { closeAfter: 10000 });
-          });
-        });
-
-    },
-
-    deleteCompany(companyToDelete) {
-      companyToDelete.destroyRecord();
-      this.get('router').transitionTo('companies');
-      setTimeout(function() {
-        window.location.reload();
```

Comment:
Ein harter Seiten-Reload sollte in einer SPA nicht nötig sein, warum passierte das hier? Ich habe dies entfernt.